draft-ietf-webpush-protocol.xml

```diff
@@ -1284,6 +1292,38 @@ HEADERS      [stream 82] +END_STREAM
           </list>
         </t>
       </section>
+      <section title="Service Name and Port Number Registration" anchor="iana.port">
+        <t>

```

Comment:
Maybe add an intro paragraph that references RFC 6335 and the registry (by name preferably).  That makes IANA happier.